api.md

```diff
@@ -274,10 +274,11 @@ Etag: "e0b2816b418b3f266309d94426ac7668ab3c1fa87798785bf82f1085cc2f6d9c"
 
 #### `POST /api/v2/domain/:domain/acl` - set ACL for a domain
 
-Sets the ACL for the given domain. HuJSON and JSON are both accepted inputs. An `If-Match` header can be set to avoid missed updates.
+Sets the ACL for the given domain.
+HuJSON and JSON are both accepted inputs.
```

Comment:
Does the request `Content-Type` need to be set either way or at all?

Reply:
`Content-Type` doesn't need to be set for requests, but the server sets it for the response.